internal/versions/versions.go

```diff
@@ -140,6 +146,35 @@ func (h *handler) GetRHCOSVersion(openshiftVersion string) (string, error) {
 	return *h.openshiftVersions[versionKey].RhcosVersion, nil
 }
 
+func (h *handler) GetMustGatherImages(openshiftVersion string, pullSecret string) (MustGatherVersion, error) {
+	versionKey, err := h.GetKey(openshiftVersion)
+	if err != nil {
+		return nil, err
+	}
+	if !h.IsOpenshiftVersionSupported(versionKey) {
+		return nil, errors.Errorf("No operators must-gather version for unsupported openshift version %s", versionKey)
+	}
+
+	releaseImage, err := h.GetReleaseImage(openshiftVersion)
+	if err != nil {
+		return nil, err
+	}
+	ocpMustGatherImage, err := h.releaseHandler.GetMustGatherImage(h.log, releaseImage, h.releaseImageMirror, pullSecret)
+	if err != nil {
+		return nil, err
+	}
+
+	mustGatherImages := make(MustGatherVersion)
```

Comment:
Maybe worth caching this map to avoid recreation on each call to GetMustGatherImages.
Or, even pre-create it for all versions so it would be initialized once. wdyt.

Reply:
caching added